### ISSUE TITLE
C# Support

### ISSUE THREAD

**Issue by aviramha** [metalbear-co/mirrord#1121]
Simple example given here https://github.com/aeb-dev/mirrord-test doesn't work with mirrord.
```
The AddressFamily InterNetwork is not valid for the System.Net.IPEndPoint end point, use InterNetworkV6 instead. (Parameter socketAddress)
```

**Comment by aviramha** [comment]
#1123 fixes an issue I faced while running the sample, but other than that everything worked for me.

Question to user:
Do you have IPv6 locally or in the cluster? also, what kind of cluster is it? managed (AKS/EKS/GKE)? remote k3s? local minikube/k3s?

**Comment by aviramha** [comment]
User solved the issue by changing listening ip to 127.0.0.1
I suspect that the application resolves localhost then binds to the addresses, binding to `[::]:port` leading to this error for some reason. (it should work afaik)
